pkgs/development/tools/misc/drush/default.nix

```diff
@@ -11,8 +12,10 @@ stdenv.mkDerivation rec {
     platforms   = platforms.all;
   };
 
-  src = fetchurl {
-    url    = "https://github.com/drush-ops/drush/archive/6.1.0.tar.gz";
+  src = fetchFromGitHub {
+    owner = "drush-ops";
+    repo  = pname;
+    rev = version;
     sha256 = "1jgnc4jjyapyn04iczvcz92ic0vq8d1w8xi55ismqyy5cxhqj6bp";
```

Comment:
I am not sure if this hash and others are still the same after switching to fetchFromGitHub.

Reply:
I'm pretty sure that this isn't the case for at least fetchgit since the first one does a `git clone` while `fetchFromGitHub` fetches a tarball from GitHub's API.

Reply:
i am currently compiling. so i will see what is the outcome

Reply:
Yep, the hash should be different, since `fetchFromGitHub` unpacks the contents of the archive.

Reply:
```suggestion
    sha256 = "sha256-0nf/m+xJmfHsFLuordiMp8UyrGGXuS70+zFHkIxLWhU=";
```

Reply:
i compiled it successfully with the given sha256sum